packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -988,6 +988,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
 
             const dataStore = await this.getDataStore(id, wait);
             const subRequest = requestParser.createSubRequest(1);
+            subRequest.url = subRequest.url.slice(1);
```

Comment:
Chatted with Si about this - I'd like to see us be consistent in request format throughout our repo (i.e. always have a leading slash if we believe that is the right behavior), so I suggested to remove this `slice()` and instead handle the leading slash in the request handler, which may require adding some back-compat support.

Reply:
I believe request handlers should be able to handle request with or without slash, but all our code that generates URIs should be consistent and always use leading slash. Note that this partially will go away with https://github.com/microsoft/FluidFramework/pull/4351, as request handling (for most part) will be by product of registering routing paths.

Reply:
Keeping this part for now. If we remove this line, it will break 2 tests in `packages\test\end-to-end-tests\src\test\primedComponent.spec.ts` : new loader, old runtime and new ContainerRuntime, old DataStoreRuntime. These two tests call the old version of request which doesn't handle the case of leading slash.